tools/app/cypress/e2e/app.cy.ts

```diff
@@ -10,76 +10,137 @@ describe('Navigation', () => {
   beforeEach(() => {
     cy.visit('http://localhost:3000');
   });
+  it('Create new base module', () => {
+    // Creates a new base module with name Basic Acceptance Test
+    cy.exec('cd ../../../&&git clone git@github.com:CyberismoCom/module-base.git&&cyberismo create project "Basic Acceptance Test" bat cyberismo-bat');
```

Comment:
This creates the base project outside of the cyberismo repo folder. That is a possible cause of conflicts. Maybe we could do this at the cyberismo repo root?